src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -10339,6 +10339,11 @@ public abstract class OSPlatformAttribute : System.Attribute
         private protected OSPlatformAttribute(string platformName) { }
         public string PlatformName { get; }
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Assembly | System.AttributeTargets.Module | System.AttributeTargets.Class | System.AttributeTargets.Struct | System.AttributeTargets.Enum | System.AttributeTargets.Constructor | System.AttributeTargets.Method | System.AttributeTargets.Property | System.AttributeTargets.Field | System.AttributeTargets.Event, AllowMultiple = true, Inherited = false)]
+    public sealed class RequiresPreviewFeaturesAttribute : System.Attribute
+    {
+        public RequiresPreviewFeaturesAttribute() { }
+    }
     [System.AttributeUsageAttribute(System.AttributeTargets.Constructor | System.AttributeTargets.Method | System.AttributeTargets.Property, Inherited=false)]
     [System.Diagnostics.ConditionalAttribute("RESOURCE_ANNOTATION_WORK")]
```

Comment:
Add a basic code-coverage test? (Look for e.g. CallerFilePathAttributeTests to what I have in mind.)